.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Version 🔖
+
+on:
+  push:
+    branches:
+      - v2
```

Comment:
```suggestion
      - change-me-to-whatever-you-want-the-release-branch-to-be
```

Reply:
just main

Reply:
You need a seperate branch to do prereleases or else you can end up in a bad state. See changeset docs where they strongly encourage it

Reply:
We could do the opposite though. Stick prereleases in alpha branch and use main for real release